includes/class-comment.php

```diff
@@ -40,6 +48,8 @@ public static function comment_reply_link( $link, $args, $comment ) {
 			return $link;
 		}
 
+		self::$remote_reply_assets_needed = true; // Request script and styles for remote reply feature.
```

Comment:
Since `build/remote-reply/index.js` depends on `.activitypub-remote-reply` being present in the DOM, it is safe to assume that the script is only needed if this filter has been triggered.

Reply:
I was initially dubious that `comment_reply_link` would run earlier than `enqueue_scripts`, but it appears to be the case!